dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/Objects.java

```diff
@@ -87,7 +87,7 @@
     MAP( "map", Map.class ),
     DASHBOARD( "dashboard", Dashboard.class ),
     PROGRAM( "program", Program.class ),
-    PROGRAMSTAGEINSTANCE( "programStageInstance", ProgramStageInstance.class ),
+    EVENT( "event", Event.class ),
```

Comment:
Hm... we have to double check that the name `PROGRAMSTAGEINSTANCE` isn't part of an generic API.

Reply:
Its part of the `/system/objectCounts` and `/dataSummary`response.

We return the count per value
* https://github.com/dhis2/dhis2-core/blob/df3d184edafc38879a3f9a099daea580bcb477b0/dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/webdomain/ObjectCount.java#L50
* https://github.com/dhis2/dhis2-core/blob/df3d184edafc38879a3f9a099daea580bcb477b0/dhis-2/dhis-services/dhis-service-reporting/src/main/java/org/hisp/dhis/datastatistics/DefaultDataStatisticsService.java#L201

I reverted the changes here for the enum name and the `value`. So we can merge the changes and wait for @mortenoh opinion if it is ok to change these public facing mentions of `programStageInstance` as well 👌🏻

Reply:
Sample from `curl -u admin:district -H 'content-type: application/json' 'https://play.dhis2.org/dev/api/system/objectCounts'`

```json
{
  "indicator": 77,
  "visualization": 292,
  "period": 384,
  "programStageInstance": 423663,
  "organisationUnit": 1332,
  "validationRule": 37,
  "dataValue": 4937367,
  "dataElement": 1036,
  "program": 14,
  "organisationUnitGroup": 18,
  "trackedEntityInstance": 73124,
  "programInstance": 73152,
  "indicatorType": 5,
  "eventVisualization": 50,
  "user": 130,
  "map": 93,
  "dataSet": 26,
  "indicatorGroup": 17,
  "dataElementGroup": 84,
  "userGroup": 34,
  "dashboard": 27
}
```